### PR TITLE
Issue #3: Setting up Basic Game Loop

### PR DESCRIPTION
**Pull request by DwangDistracted**
**ISSUES:** 3, 5, 11

The Basic Game loop is set up. Please Review and Comment.

1: The Main Class queries the LevelLoader for the next level.
2: The Main Class creates a Game Object with the Level Info from Level Loader
3: The Game Object enters a loop of Player Turn -> Zombie Turn -> until player says quit.

The Turn Engine as a separate class is removed and integrated into Game.

The LevelInfo Class (sans serialization has been set up.

### CODE REVIEW

File: src/util/Logger.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package util;
```
Comment (dershao) [inline src/util/Logger.java]: You probably know this already but in case you were unaware, there's already a logger in Java (java.util.logging)

Comment (DwangDistracted) [reply src/util/Logger.java]: I am a massive idiot

File: src/levels/LevelInfo.java [hunk 2]
```diff
@@ -1,20 +1,85 @@
...
-	private short gridX;
-	private short gridY;
+	private int gridX;
+	private int gridY;
+	
+	private int initResources;
+	private int resPerTurn;
+	
+	private HashMap<Zombie, Integer> zombies;
+	private List<Plant> prohibitedPlants;
```
[the remark this answers is not in the document]
Comment (DwangDistracted) [reply src/levels/LevelInfo.java]: Plants that can't be used on a level

File: bin/.gitignore [hunk 3]
```diff
@@ -1,5 +1,3 @@
-/Main.class
```
[the remark this answers is not in the document]
Comment (DwangDistracted) [reply bin/.gitignore]: Now set up to ignore all class files

**Comment by dershao** [comment]
Good job, approved.

I think some .class files have been committed to the git history though. You'll have to delete them and re-commit before merging.

**Comment by DwangDistracted** [comment]
Closed due to outdated

**Pull request closed without merging**